packages/web/src/store/wallet/sagas.ts

```diff
@@ -217,6 +223,30 @@ function* fetchBalanceAsync() {
   }
 }
 
+/**
+ * Check if we can send WAudio to a recipient by checking if they already have
+ * an associated WAudio token account, or if they have enough SOL to create one.
+ */
+function* checkAssociatedTokenAccountOrSol() {
+  const walletClient = yield* getContext('walletClient')
+  const ret = yield* select(getSendData)
```

Comment:
We should be able to get the wallet address directly from the action (as an argument to this saga) without having to depend on the reducer already having run